libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/tabs/reviewCreateTab.tsx

```diff
@@ -142,14 +141,12 @@ export const reviewCreateTab = (
   shouldClearDetails: boolean,
   onCreateClick: () => void,
   {
-    workflowName,
     isCreating,
     isCreated,
```

Comment:
If we auto direct users, this "isCreated" - is this meaningful?

Reply:
that is what my confusion was, why do we need isCreated

Reply:
we needed it because we weren't auto directing users; we wanted to display different behaviors